packages/react/src/components/ColorDropdown/ColorDropdown.jsx

```diff
@@ -47,8 +48,14 @@ const propTypes = {
   translateWithId: PropTypes.func,
   /** The selected color, use to set initial color */
   selectedColor: colorPropType,
+  // TODO: remove deprecated testID in v3.
+  // eslint-disable-next-line react/require-default-props
+  testID: deprecate(
```

Comment:
I believe this component is still experimental so we can probably swap out ID without deprecate.